src/index.js

```diff
@@ -386,7 +383,7 @@ class Dropzone extends React.Component {
     customProps.forEach(prop => delete divProps[prop])
 
     return (
-      <div
+      <props.tag
```

Comment:
The syntax is quite misleading IMO. In that case I'd prefer `createElement` rather than JSX.

Also I'm doing some work in https://github.com/react-dropzone/react-dropzone/pull/468 that will make this simpler so if you could help me with this one instead it would be cool!